src/base/bittorrent/torrentinfo.cpp

```diff
@@ -122,7 +122,7 @@ nonstd::expected<TorrentInfo, QString> TorrentInfo::loadFromFile(const Path &pat
     }
     catch (const std::bad_alloc &e)
     {
-        return nonstd::make_unexpected(tr("Torrent file read error: %1").arg(e.what()));
+        return nonstd::make_unexpected(tr("Torrent file read error: %1").arg(QString::fromUtf8(e.what())));
```

Comment:
I know that `QString::fromUtf8()` was actually used here, but still, shouldn't `QString::fromLocal8Bit()` used here?

Reply:
I think you are right here, fixed.